corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -1146,28 +1150,31 @@ def get_ledger_value(case_id, section_id, entry_id):
             raise LedgerValueNotFound
 
     @staticmethod
-    def save_ledger_values(ledger_values, deprecated_form=None):
-        if not ledger_values:
+    def save_ledger_values(ledger_values, stock_result=None):
```

Comment:
no but it didn't have enough info so passed in the full stock result so that I could get the relevant case IDs affected by the deprecated form